server/src/main/java/org/apache/gravitino/server/web/rest/TableOperations.java

```diff
@@ -99,10 +116,20 @@ public Response listTables(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "create-table." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "create-table", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          " ((METALAKE::USE_CATALOG || CATALOG::USE_CATALOG || SCHEMA::USE_CATALOG) && "
+              + "((METALAKE::USE_SCHEMA || CATALOG::USE_SCHEMA || SCHEMA::USE_SCHEMA) "
+              + "&& (METALAKE::CREATE_TABLE || CATALOG::CREATE_TABLE || "
+              + "SCHEMA::CREATE_TABLE) || SCHEMA::OWNER)) || "
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files